src/components/Button/Button.module.css

```diff
@@ -11,49 +11,28 @@
  * 4) Adds a gap between children to automatically handle spacing between icons and text
  */
 .button {
+  @mixin eds-theme-typography-button-label;
   display: inline-flex;
   align-items: center;
   justify-content: center;
   margin: 0; /* 3 */
   gap: 0.75rem; /* 4 */
   border-width: var(--eds-theme-border-width);
   border-style: solid;
-  border-color: var(--eds-theme-color-primary-border);
   border-radius: var(--eds-theme-border-radius);
-  color: var(--eds-theme-color-primary-foreground);
-  background-color: transparent;
   cursor: pointer;
   transition: all var(--eds-anim-fade-quick) var(--eds-anim-ease);
 
-  /* TODO: make default color prop in Icon inherit or something? */
-  svg {
-    fill: currentColor;
-  }
-
-  &:hover {
-    color: var(--eds-theme-color-primary-foreground-hover);
-    border-color: var(--eds-theme-color-primary-border-hover);
-  }
-
   &:focus {
-    background-color: transparent;
-  }
-
-  &:focus-visible {
```

Comment:
`focus-visible` is used so the focus outline only appears when a user tabs to a button and not when a user clicks on a button. This is theoretically fine, but I don't want this in here unless we're sure we don't need to do anything else to make sure keyboard users do in fact get the outline. I believe `Along` is using a polyfill for this, and I don't know what the ramifications are of not using that (it might just be that Internet Explorer doesn't get the same functionality).

Reply:
mm yeah good call. we should check browser support for `focus-visible` now (at one point I thought it was only Chrome)